test/setCommand_test.js

```diff
@@ -0,0 +1,155 @@
+goog.module('datalayerhelper.helper.testing.setCommand');
+goog.setTestOnly();
+
+const {DataLayerHelper} = goog.require('helper');
+
+describe('The set command', () => {
+  let dataLayer;
+  let dlh;
+  let commandAPI;
+  beforeAll(() => {
+    dataLayer = [];
+    commandAPI = function() {
```

Comment:
Can this be an arrow function too?

Reply:
So there is a reference to the arguments object in the commandAPI function, and arguments is not supported by arrow functions, here's some documentation on that if you're interested https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments#Description